### PR TITLE
#166789966 User Should Opt In And Out Of Notifications:

### PR DESCRIPTION
**Pull request by elemanhillary-zz**
#### What does this PR do?
- adds button to turn off and on notifications

#### Description of Task to be completed
- enable user to turn off and on notifications

#### How should this be manually tested?
```bash
git clone https://github.com/andela/ah-92explorers-frontend.git
cd ah-92explorers-frontend
git checkout ft-notifications-optin-optout-166789966
npm i
npm run dev
```
- Navigate to `localhost:<port>/profile
- toggle on and off the notification button

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- [#168426852](https://www.pivotaltracker.com/story/show/168426852)

#### Screenshots (if appropriate)
Imgur

#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by kodek-sleuth — COMMENTED** [review]
This feature's functionality is to be fully seen when the feature `Users should be able to get Notifications` is done otherwise its just a button that toggles

**Review by niyoceles — APPROVED** [review]

**Review by Hervera — APPROVED** [review]
I do not see a notification message while Opt In And Out Of Notifications. How can an end user be sure that an action has been successful?

**Review by niyoceles — DISMISSED** [review]
@elemanhillary I can't see the alert message for turn on or turn off the notification, you should display them

**Review by Hervera — DISMISSED** [review]
I do not see a notification message while Opt In And Out Of Notifications. How can an end user be sure that an action has been successful?

**Review by kodek-sleuth — DISMISSED** [review]
According to the importance of this feature to start another and the explanation given. I think this is ready to be merged

**Review by kodek-sleuth — DISMISSED** [review]

**Review by niyoceles — DISMISSED** [review]
The feedback has been implemented well

**Review by IsaiahRn — DISMISSED** [review]
It can be merged!

**Review by kodek-sleuth — DISMISSED** [review]
I subscribed for notifications on the homepage, then I went to read an article on another page and I still realised that my subscription had toggled off

**Comment by elemanhillary-zz** [comment]
> I subscribed for notifications on the homepage, then I went to read an article on another page and I still realised that my subscription had toggled off

the best way to put a` bug` label on a PR is you should reproduce the error in question and let it be confirmed by others.

File: .huskyrc.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "hooks": {
-    "pre-commit": "npm run lint && npm test"
```
Comment (niyoceles) [inline .huskyrc.yml]: Kindly don't remove this`npm test' in .huskyrc.myl

**Review by niyoceles — DISMISSED** [review]

**Review by niyoceles — APPROVED** [review]
Feedback implemented well.

**Review by Hervera — APPROVED** [review]
well done! I can see the Opt In or Out of notifications works on every page

**Review by IsaiahRn — APPROVED** [review]
Well done! I can be merged

**Review by kodek-sleuth — APPROVED** [review]

**Pull request merged**